src/tck/src/TckServer.cc

```diff
@@ -143,7 +145,9 @@ std::string TckServer::handleJsonRequest(std::string_view request)
     // If the JSON request is an object, then it's a single request.
     if (jsonRequest.is_object())
     {
+      std::cout << "Handling request " << jsonRequest.dump() << std::endl;
```

Comment:
Will we at some point introduce some logging tool to the TCK implementation?

Reply:
We probably should... I just kept these here so there'd be some indication that the server received a request, processed it, and sent back a response to the TCK